File: src/parsers/java/TestSimple.java
package com.example.simple;

import java.util.List;
import java.util.ArrayList;

/**
 * Simple test file for Java to XLang parser
 */
public class TestSimple {
    private int value;
    private static final String CONSTANT = "Hello";

    public TestSimple(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }

    public void setValue(int value) {
        this.value = value;
    }

    public static int add(int a, int b) {
        return a + b;
    }

    public static void main(String[] args) {
        TestSimple obj = new TestSimple(42);
        int result = add(obj.getValue(), 10);
        System.out.println("Result: " + result);

        // Test control flow
        if (result > 50) {
            System.out.println("Greater than 50");
        } else {
            System.out.println("Less than or equal to 50");
        }

        // Test loop
        for (int i = 0; i < 5; i++) {
            System.out.println("Count: " + i);
        }

        // Test enhanced for loop
        List<String> items = new ArrayList<>();
        items.add("apple");
        items.add("banana");
        items.add("cherry");

        for (String item : items) {
            System.out.println("Item: " + item);
        }
    }
}

File: src/parsers/kotlin/TestSimple.kt
package com.example

// Simple test file for Kotlin parser

fun main() {
    println("Hello, World!")
    val x = 42
    var y = "test"

    if (x > 10) {
        println("x is greater than 10")
    } else {
        println("x is 10 or less")
    }

    for (i in 1..5) {
        println(i)
    }
}

class SimpleClass(val name: String) {
    fun greet(): String {
        return "Hello, $name!"
    }
}
